packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -841,6 +855,13 @@ class SliverAppBar extends StatefulWidget {
   /// Defaults to being adapted to the current [TargetPlatform].
   final bool centerTitle;
 
+  /// The margin between [title] and [leading] content. This margin is applied
+  /// even if there is no leading content. If you want [title] to take all the
+  /// space available, set this value to 0.0.
+  ///
+  /// Defaults to the specification in [NavigationToolbar].
+  final double titleMargin;
```

Comment:
(similar comments to the other place with this member)

Reply:
Fixed.